Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -464,6 +458,20 @@ public class PBXProjGenerator {
         return pbxtarget
     }
 
+    func getInfoPlist(_ spec: ProjectSpec, _ sources: [TargetSource]) -> Path? {
+        return sources
+            .lazy
+            .map { spec.basePath + $0.path }
+            .flatMap { (path) -> Path? in
+                if path.isFile {
+                    return path.lastComponent == "Info.plist" ? path : nil
+                } else {
+                    return path.first(where: { $0.lastComponent == "Info.plist" })
```

Comment:
Does this look in direct children or all recursive children? If the former it would be a regression as files in sub folders won't be found anymore. It's an edge case, but I have projects that rely on this

Reply:
Yep. PathKit uses [`FileManager.enumerator(at URL:)`](https://developer.apple.com/documentation/foundation/nsfilemanager/1409571-enumeratoraturl?language=objc) which does "deep" aka recursive enumeration.